examples/auction/src/lib.rs

```diff
@@ -133,7 +142,16 @@ fn auction_bid<S: HasStateApi>(
     let previous_balance = balance - amount;
 
     // Ensure that the new bid exceeds the highest bid so far
-    ensure!(amount > previous_balance, BidError::BidTooLow);
+    ensure!(amount > previous_balance, BidError::BidBelowCurrentBid);
+
+    // Calculate the difference between the previous bid and the new bid in CCD.
+    let amount_difference = amount - previous_balance;
+    // Get the current exchange rate used by the chain
+    let exchange_rates = host.exchange_rates();
+    // Convert the CCD difference to EUR
+    let euro_cent_difference = exchange_rates.convert_amount_to_euro_cent(amount_difference);
+    // Ensure that the bid is at least 100 EUR cent more than the previous bid
```

Comment:
Where does the 100 euro cent come from? Isn't it just that the difference is larger than `minimum_raise`?